### ISSUE TITLE
ASP.NET Core WebAPI includes InvariantGlobalization setting in .csproj in .NET 8 but is not stated as a change.

### ISSUE THREAD

**Issue by patr543a** [dotnet/aspnetcore#52319]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

`<InvariantGlobalization>true</InvariantGlobalization>` is included in the .csproj file in ASP.NET Core WebAPI by default in .NET 8 only was never stated as a change.

### Expected Behavior

`<InvariantGlobalization>true</InvariantGlobalization>`  should not be in the .csproj by default as is not the case with .NET 7 only .NET 8 and is never ever stated in the changes which gives me the impression this is a bug. This causes issues with Entity Framework Core which i experienced and it took hours before i knew this was the cause because it was never stated as a change at all

### Steps To Reproduce

- Open Visual Studio 2022 Preview
- Create a new project of ASP.NET Core WebAPI
- Open .csproj file
- `<InvariantGlobalization>true</InvariantGlobalization>` will be in it

### Exceptions (if any)

Doesn't really give any exceptions but will cause exceptions like `CultureNotFound` in Entity Framework Core.

### .NET Version

8.0.100

### Anything else?

IDE: Version 17.9.0 Preview 1.0
Output of `dotnet --info`
```
.NET SDK:
 Version:           8.0.100
 Commit:            57efcf1350
 Workload version:  8.0.100-manifests.71b9f198

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19045
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.100\

.NET workloads installed:
 Workload version: 8.0.100-manifests.71b9f198
 [aspire]
   Installation Source: VS 17.9.34310.174
   Manifest Version:    8.0.0-preview.1.23557.2/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.0.0-preview.1.23557.2\WorkloadManifest.json
   Install Type:              Msi

 [maui-windows]
   Installation Source: VS 17.9.34310.174, VS 17.7.34221.43
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Install Type:              Msi

 [maccatalyst]
   Installation Source: VS 17.9.34310.174, VS 17.7.34221.43
   Manifest Version:    17.0.8478/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maccatalyst\17.0.8478\WorkloadManifest.json
   Install Type:              Msi

 [ios]
   Installation Source: VS 17.9.34310.174, VS 17.7.34221.43
   Manifest Version:    17.0.8478/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.ios\17.0.8478\WorkloadManifest.json
   Install Type:              Msi

 [android]
   Installation Source: VS 17.9.34310.174, VS 17.7.34221.43
   Manifest Version:    34.0.43/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.android\34.0.43\WorkloadManifest.json
   Install Type:              Msi

 [maui-maccatalyst]
   Installation Source: VS 17.7.34221.43
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Install Type:              Msi

 [maui-ios]
   Installation Source: VS 17.7.34221.43
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Install Type:              Msi

 [maui-android]
   Installation Source: VS 17.7.34221.43
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Install Type:              Msi


Host:
  Version:      8.0.0
  Architecture: x64
  Commit:       5535e31a71

.NET SDKs installed:
  3.1.401 [C:\Program Files\dotnet\sdk]
  7.0.403 [C:\Program Files\dotnet\sdk]
  8.0.100 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.21 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.21 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.24 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.21 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.24 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.7 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.24 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  x86   [C:\Program Files (x86)\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x86\InstallLocation]

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
```

**Comment by gfoidl** [comment]
For reference: https://github.com/dotnet/aspnetcore/pull/47066 made that change.

**Comment by patr543a** [comment]
> For reference: #47066 made that change.

I see but it still does not say anything about it in the documentation. So i guess i'll have to go report it there

**Comment by gfoidl** [comment]
@patr543a just wait a bit...the issue can be moved to docs also, no need to create there a separate one (at the moment).
My comment was solely about giving a reference to the PR that made that change, so triaging this issue may become easier (others don't have to search for the PR).

@eerhardt can you please chime in?

**Comment by patr543a** [comment]
Since it seams like nothing seams to be happening here i made a documentation issue post. (I know i was told not to)
You can find the documentation issue post [here](https://github.com/dotnet/docs/issues/38438#issue-2011784087)

**Comment by eerhardt** [comment]
This was an intentional change to the web api template. But it wasn't intentional to leave it out of the docs. Thanks for opening the docs issue, @patr543a.

**Comment by eerhardt** [comment]
> This was an intentional change to the web api template.

After more investigation, this setting was only supposed to be enabled when the template was created for Native AOT mode. See #48238. Looks like we missed a few spots in that PR. Re-opening to fix them.

**Comment by eerhardt** [comment]
Re-opening to track backporting to 8.0.

**Comment by eerhardt** [comment]
@dotnet-policy-service, this isn't a PR.

But thanks for the reminder. The PR (#52461) has been merged into release/8.0. Closing.
